docs/src/main/asciidoc/logging.adoc

```diff
@@ -741,7 +741,7 @@ implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
 
 === Use MDC to add contextual log information
 
-Quarkus overrides log MDC (Mapped Diagnostic Context) to improve the compatibility with its reactive core.
+Quarkus overrides log Mapped Diagnostic Context (MDC) to improve the compatibility with its reactive core.
```

Comment:
"Quarkus overrides log Mapped Diagnostic Context" sounds quite odd to me

Could we say

"Quarkus overrides the logging Mapped Diagnostic Context" instead?

Reply:
Agree, that sounds better.

Reply:
Done and done.